torch/testing/_internal/common_methods_invocations.py

```diff
@@ -12265,6 +12272,15 @@ def ref_pairwise_distance(input1, input2):
            # https://github.com/pytorch/pytorch/issues/66357
            check_batched_forward_grad=False,
            sample_inputs_func=sample_inputs_transpose_swapdims),
+    OpInfo('transpose_copy',
+           ref=lambda x, dim0, dim1: ref_transpose(x, dim0, dim1).copy(),
+           # transpose_copy does not support automatic differentiation for outputs with complex dtype
+           dtypes=all_types_and(torch.bool, torch.bfloat16, torch.half),
+           dtypesIfCUDA=all_types_and(torch.bool, torch.bfloat16, torch.half),
```

Comment:
Probably `dtypesIfCUDA` is bot needed if it's the same as `dtypes`?

Reply:
Removed dtypesIfCUDA as unnecessary.